packages/x-charts/src/ScatterChart/seriesConfig/tooltip.ts

```diff
@@ -22,4 +22,6 @@ const tooltipGetter: TooltipGetter<'scatter'> = (params) => {
   };
 };
 
-export default tooltipGetter;
+export const axisTooltipGetter: AxisTooltipGetter<'scatter', 'x' | 'y'> = (series) => {
+  return Object.values(series).map((s) => ({ direction: 'x', axisId: s.xAxisId }));
```

Comment:
This only works if the axis has a `data` property, which the scatter chart doesn't. I'll see if there's any other way we can fix this.

Reply:
However, when using composition, we can create scatter series and add a `data` property to the x-axis, in which case this would be possible 🤔

Reply:
If we remove the `, data: [1, 2]` from the test above it doesn't work, shouldn't it work in both cases?